packages/cli/create-medusa-app/src/utils/create-db.ts

```diff
@@ -16,6 +16,12 @@ export default async function createDb({ client, db }: CreateDbOptions) {
   await client.query(`CREATE DATABASE "${db}"`)
 }
 
+async function doesDbExist (client: pg.Client, dbName: string): Promise<boolean> {
+  const result = await client.query(`SELECT datname FROM pg_catalog.pg_database WHERE datname='${dbName}';`)
+
+  return !!result.rowCount
+}
+
```

Comment:
Does this query work fine? Because from my experience, the PG client throws an exception before running the query (in case the database is missing). 

This is how I handled it within the medusa monorepo.

https://github.com/medusajs/medusa/pull/8718/files#diff-0cd2ae0bc94a653c4b2dff2733a3dd0a2efea552d2ae9ca6a36b1fe28c0be600R96-R106

Reply:
Worked fine in my testing 😄 I'll generate a snapshot and maybe you can give it a go?

Reply:
Sure. Maybe in my case I was relying on the Mikro ORM so that is performing some checks.

Reply:
@thetutlage seems like the snapshot isn't working. Can you go to `packages/cli/create-medusa-app` and run `yarn dev` to test it out? You can also use the `--directory-path` option to create the project in a different directory. For example, `yarn dev --directory-path ~/test`.

Reply:
Sure. Lemme do it now!

Reply:
Yes. That works. So in my case the issue is from the Mikro ORM then